DESCRIPTION

```diff
@@ -1,11 +1,12 @@
 Package: drat
 Type: Package
 Title: 'Drat' R Archive Template
-Version: 0.1.6
-Date: 2020-05-29
+Version: 0.1.6.4
+Date: 2020-06-09
 Author: Dirk Eddelbuettel with contributions by Carl Boettiger, Neal Fultz,
  Sebastian Gibb, Colin Gillespie, Jan Górecki, Matt Jones, Thomas Leeper,
- Steven Pav, Jan Schulz, Christoph Stepper and Felix G.M. Ernst. 
+ Steven Pav, Jan Schulz, Christoph Stepper, Felix G.M. Ernst and Patrick 
+ Schratz. 
```

Comment:
Similar. My project. At the end of the day shouldn't I decide whom I anoint as a contributor?  No need to change now, but please consider it next time.

Reply:
I totally agree. However, I didn't want to highjack Patrick's idea. It was his and he did the hard work of coming up with it.